lib/rules/selector-type-no-unknown/index.js

```diff
@@ -89,6 +89,7 @@ const rule = function (actual, options) {
             || svgTags.indexOf(tagNameLowerCase) !== -1
             || keywordSets.nonStandardHtmlTags.has(tagNameLowerCase)
             || mathMLTags.indexOf(tagNameLowerCase) !== -1
+            || tagNameLowerCase === "/deep/"
```

Comment:
Let's move this to the [`isStandardSyntaxTypeSelector` util](https://github.com/stylelint/stylelint/blob/master/lib/utils/isStandardSyntaxTypeSelector.js). It's where we build on top of `postcss-selector-parser`.

Also, let's return `false` for all [_reference combinators_](https://www.w3.org/TR/2013/WD-selectors4-20130502/#idref-combinators), not just `/deep/` i.e. return `false` for anything that starts and ends with a `/`.

Be sure to comment the line e.g.
```
// Reference combinators like `/deep/`
```